lib/logstash/inputs/kafka.rb

```diff
@@ -287,7 +292,10 @@ def extract_metadata_level(decorate_events_setting)
 
   public
   def run(logstash_queue)
-    @runner_consumers = consumer_threads.times.map { |i| subscribe(create_consumer("#{client_id}-#{i}")) }
+    @runner_consumers = consumer_threads.times.map do |i|
+      thread_group_instance_id = consumer_threads > 1 && group_instance_id ? "#{group_instance_id}-#{i}" : group_instance_id
```

Comment:
Any thoughts on always adding "i" to the name? this has an upside and downside..downside is that we're not using the exact name the user set when threads = 1, the upside is that naming is consistent regardless the number of threads, and also consistent with client_id naming.

Reply:
I had the same thought, but given that `consumer_threads` is 1 by default, I thought not adding it, was the minimal surprising behavior for users.